client/client.go

```diff
@@ -5,6 +5,12 @@ import (
 	wcm "github.com/voidshard/wysteria/common/middleware"
 )
 
+const (
+	FACET_COLLECTION = "collection"
+	FACET_ITEM_TYPE = "itemtype"
+	FACET_ITEM_VARIANT = "variant"
```

Comment:
Thanks! I keep crossing over between Go & Python .. my brain leaves behind the idioms.